corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js

```diff
@@ -124,6 +124,10 @@ FormplayerFrontend.on('startForm', function (data) {
     FormplayerFrontend.request("clearMenu");
     FormplayerFrontend.Menus.Util.showBreadcrumbs(data.breadcrumbs);
 
+    var urlObject = Util.currentUrlToObject();
+    urlObject.setSessionId(data.session_id);
+    Util.setUrlToObject(urlObject);
```

Comment:
i think it had something to do with one question per screen. i vaguely remember this too